src/inverseRobot.js

```diff
@@ -8,6 +8,17 @@
 
 function inverseRobot(robot) {
   // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (newRobot[robot[key]] !== undefined) {
```

Comment:
You're checking if the value of newRobot key isn't empty, instead you should compare if the newRobot object doesn't already have a similar key

Reply:
in this case i agree with you, and new if statement   `if (newRobot[robot[key]]) ` will check if any of the input object values are repeated in output object.